### PR TITLE
Update Go/Docker dependencies, release 1.5.4

### PR DESCRIPTION
**Pull request by grantbeattie**
- Upgraded Go from 1.18 to 1.20.
- Upgraded base docker image to alpine:3.18.2.
- Upgraded the AWS SDK KMS component to v1.24.1.
- Upgraded the AWS SDK config component to v1.18.32.
- Upgraded the Google/uuid library to v1.3.0.
- Upgraded the urfave/cli library to v1.22.14.

Release shush 1.5.4.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # CHANGES
 
+## 1.5.4
+
+- Upgraded Go from 1.18 to 1.20.
+- Upgraded base docker image to alpine:3.18.2.
+- Upgraded the AWS SDK KMS component to v1.24.1.
+- Upgraded the AWS SDK config component to v1.18.32.
+- Upgraded the Google/uuid library to v1.3.0.
+- Upgraded the urfave/cli library to v1.22.14.
```
Comment (djmattyg007) [inline CHANGES.md:10]: Do people care about individual dependency updates? I think people would care about the version of Golang and the version of the core AWS SDK components in use, but probably not a UUID library?

File: go.sum [hunk 2]
```diff
@@ -1,47 +1,91 @@
...
 github.com/urfave/cli v1.22.2 h1:gsqYFH8bb9ekPA12kRo0hfjngWQjkJPlN9R0N78BoUo=
 github.com/urfave/cli v1.22.2/go.mod h1:Gos4lmkARVdJ6EkW0WaNv/tZAAMe9V7XWyB60NtXRu0=
+github.com/urfave/cli v1.22.14 h1:ebbhrRiGK2i4naQJr+1Xj92HXZCrK7MsyTS/ob3HnAk=
+github.com/urfave/cli v1.22.14/go.mod h1:X0eDS6pD6Exaclxm99NJ3FiCDRED7vIHpx2mDOHLvkA=
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
 gopkg.in/yaml.v2 v2.2.2/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
 gopkg.in/yaml.v2 v2.2.8/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
+gopkg.in/yaml.v2 v2.4.0/go.mod h1:RDklbk79AGWmwhnvt/jBztapEOGDOx6ZbXqjP6csGnQ=
+gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
+gopkg.in/yaml.v3 v3.0.1/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
```
Comment (djmattyg007) [inline go.sum:91]: It might be worth running `go mod tidy` to clean up this list a little bit.

File: CHANGES.md (hunk 1 above)
Comment (grantbeattie) [reply CHANGES.md:10]: probably not, I just included everything for completeness.

File: go.sum (hunk 2 above)
Comment (grantbeattie) [reply go.sum]: Yes, thank you, have done so now :)

**Pull request merged**